ui/src/components/DenylistForm/index.jsx

```diff
@@ -44,7 +45,7 @@ export default class DenylistForm extends Component {
     /** Callback function fired when an address is created. */
     onAddressAdd: func.isRequired,
     /** Callback function fired when an address is deleted. */
-    onAddressDelete: func,
+    onAddressDelete: func.isRequired,
```

Comment:
Remove `onAddressDelete` from `static defaultProps` since it is set as required.